src/npt/arch/aarch64.rs

```diff
@@ -16,10 +17,10 @@ bitflags::bitflags! {
         /// (not a 2M, 1G block)
         const NON_BLOCK =   1 << 1;
         /// Memory attributes index field.
-        const ATTR_INDX =   0b111 << 2;
+        const ATTR =   0b1111 << 2;
         /// Non-secure bit. For memory accesses from Secure state, specifies whether the output
         /// address is in Secure or Non-secure memory.
-        const NS =          1 << 5;
+        // const NS =          1 << 5;
```

Comment:
Why delete this

Reply:
![image](https://github.com/user-attachments/assets/8e7fd433-943c-4a46-aa46-7e5fceac5a68)
there is no NS attributes in stage 2

Reply:
Add more annotations here.

Will this flag be available again after SEL2 is supported?

Reply:
My bad，there is still NS bit in stage 2，but it is in the 55th bit，and we have modify it .